packages/grafana-data/src/utils/featureToggle.ts

```diff
@@ -0,0 +1,16 @@
+import { BootData } from '../types';
+
+declare global {
+  interface Window {
+    grafanaBootData?: BootData;
+  }
+}
+
+/**
+ * Grafana data does not have access to runtime, so we are accessing the window object to get the feature toggles.
```

Comment:
Could you please rewrite this comment to what this API does and how to use it and not why we have the API?

Consider that other developers (external ones too) will explore the API through this comments and they should be treated as documentation.